### PR TITLE
FDWs: install restructure

### PR DESCRIPTION
**Pull request by drothery-edb**

### CODE REVIEW

File: product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/01_hadoop_rhel8_x86.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "RHEL 8 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/01_hadoop_rhel8_x86.mdx:2]: I think it should be  "RHEL 8/OL 8 on x86_64"

File: product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/03_hadoop_rhel7_x86.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: "RHEL 7 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/03_hadoop_rhel7_x86.mdx:2]: I think it should be "RHEL 7/OL 7 on x86_64"

File: product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/index.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Installing the Hadoop Foreign Data Wrapper"
+---
+
+For platform-specific install instructions, see:
+
+- Linux x86-64 (amd64):
+
+  - [RHEL 8](01_hadoop_rhel8_x86)
```
Comment (nidhibhammar) [inline product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/index.mdx:9]: [RHEL 8/ OL 8]

File: product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/index.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  - [Rocky Linux 8/AlmaLinux 8](02_hadoop_other_linux8_x86)
+  - [RHEL 7](03_hadoop_rhel7_x86)
```
Comment (nidhibhammar) [inline product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter/index.mdx:12]: [RHEL 7/OL 7]

File: product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/01_mongo_rhel8_x86.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: "RHEL 8 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/01_mongo_rhel8_x86.mdx:2]: RHEL 8/OL 8

File: product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/03_mongo_rhel7_x86.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+title: "RHEL 7 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/03_mongo_rhel7_x86.mdx:2]: RHEL 7/OL 7

File: product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx [hunk 7]
```diff
@@ -12,12 +12,18 @@ The MongoDB Foreign Data Wrapper is supported on the following platforms:
 
 **Linux x86-64**
 
- -   RHEL 8/Rocky Linux 8/AlmaLinux 8/OL 8
- -   RHEL 7/CentOS 7/OL 7
+ -   RHEL 8
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx:15]: RHEL 8/OL 8

File: product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx [hunk 8]
```diff
@@ -12,12 +12,18 @@ The MongoDB Foreign Data Wrapper is supported on the following platforms:
...
+ -   RHEL 7
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx:16]: RHEL 7/OL 7

File: product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx [hunk 9]
```diff
@@ -12,12 +12,18 @@ The MongoDB Foreign Data Wrapper is supported on the following platforms:
...
+ -   Rocky Linux 8/AlmaLinux 8/OL 8
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx:17]: remove OL 8

File: product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx [hunk 10]
```diff
@@ -12,12 +12,18 @@ The MongoDB Foreign Data Wrapper is supported on the following platforms:
...
+ -   CentOS 7/OL 7
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx:18]: OL 7

File: product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/index.mdx [hunk 11]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Installing the MongoDB Foreign Data Wrapper"
+---
+
+For platform-specific install instructions, see:
+
+- Linux x86-64 (amd64):
+
+  - [RHEL 8](01_mongo_rhel8_x86)
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/index.mdx:9]: RHEL 8/OL 8

File: product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/index.mdx [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+  - [Rocky Linux 8/AlmaLinux 8](02_mongo_other_linux8_x86)
+  - [RHEL 7](03_mongo_rhel7_x86)
```
Comment (nidhibhammar) [inline product_docs/docs/mongo_data_adapter/5.3.0/04_installing_the_mongo_data_adapter/index.mdx:12]: RHEL 7/OL 7

File: product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/01_mysql_rhel8_x86.mdx [hunk 13]
```diff
@@ -0,0 +1,63 @@
+---
+title: "RHEL 8 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/01_mysql_rhel8_x86.mdx:2]: RHEL 8/OL 8

File: product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/02_mysql_other_linux8_x86.mdx [hunk 14]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Rocky Linux 8/AlmaLinux8 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/02_mysql_other_linux8_x86.mdx:2]: Rocky Linux 8/AlmaLinux 8 on x86_64

File: product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/03_mysql_rhel7_x86.mdx [hunk 15]
```diff
@@ -0,0 +1,75 @@
+---
+title: "RHEL 7 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/03_mysql_rhel7_x86.mdx:2]: RHEL 7/OL 7

File: product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/index.mdx [hunk 16]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Installing the MySQL Foreign Data Wrapper"
+---
+
+For platform-specific install instructions, see:
+
+- Linux x86-64 (amd64):
+
+   -   [RHEL 8](01_mysql_rhel8_x86)
```
Comment (nidhibhammar) [inline product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/index.mdx:9]: RHEL 8/OL 8

File: product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/index.mdx [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+
+   -   [Rocky Linux 8/AlmaLinux8](02_mysql_other_linux8_x86)
+   -   [RHEL 7](03_mysql_rhel7_x86)
```
Comment (nidhibhammar) [inline product_docs/docs/mysql_data_adapter/2.7.0/04_installing_the_mysql_data_adapter/index.mdx:12]: RHEL 7/OL 7

File: product_docs/docs/mysql_data_adapter/2.7.0/02_requirements_overview.mdx [hunk 18]
```diff
@@ -41,4 +41,4 @@ The MySQL Foreign Data Wrapper is certified with EDB Postgres Advanced Server 10
```
Comment (nidhibhammar) [inline product_docs/docs/mysql_data_adapter/2.7.0/02_requirements_overview.mdx:41]: Can we have the list of platforms in sync with other data adapters?

File: product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx (hunk 10 above)
Comment (drothery-edb) [reply product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx:18]: You meant I should remove OL 7 from line 18, correct?

**Review by nidhibhammar — APPROVED** [review]
Looks good to me

**Pull request merged**